### ISSUE TITLE
[A11y] Desktop/Mobile Collection Landing: All/Any Dropdown Menu

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1879]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Collection Show

### Admin user?

No

### Issue Description

Collection Landing: https://staging.oregondigital.org/collections/gwilliams?locale=en

The All/Any dropdown menu is missing accessible name.

### Suggested Fix

Add accessible name

### Can anybody QA or does this need an expert?

A11y familiarity recommended

**Comment by CGillen** [comment]
dupe #1822 
Fixed by #1878 & #1911
